app/assets/javascripts/katello/smart_proxies/capsule_status.js

```diff
@@ -0,0 +1,28 @@
+function loadPulp() {
+    var item = $('#pulp-status'),
+        version = $('#pulp-version'),
+        url = item.data('url');
```

Comment:
`data-ajax-url` loads a partial via ajax. If I understand correctly, this passes a json to generateItem which renders a simple icon with data from the json in tooltip?